examples/nft_improved_worker/threejs_worker_A_Flamingo_gltf.js

```diff
@@ -208,8 +208,10 @@ function start( container, marker, video, input_width, input_height, canvas_draw
             // other different images, possibly with different aspect ratio
             if (!window.firstPositioning) {
                 window.firstPositioning = true;
-                model.position.y = (msg.width / msg.dpi) * 1000 / objPositions.width;
-                model.position.x = (msg.height / msg.dpi) * 1000 / objPositions.height;
+                // model.position.y = (msg.width / msg.dpi) * 1000 / objPositions.width;
+                // model.position.x = (msg.height / msg.dpi) * 1000 / objPositions.height;
+                model.position.y = (msg.width / msg.dpi * 2.54 * 10)/2.0;
```

Comment:
we use the value 2.54 because we want to convert inch to cm

Reply:
Thanks, I was about to ask you :D

Reply:
@kalwalt wrote it down as a comment on the code, about this